File: java/strings/Truncate_Sentence_1816.java
package java.strings;

public class Truncate_Sentence_1816 {
    public String truncateSentence(String s, int k) {
        String[] words=s.split(" ");
        StringBuilder res=new StringBuilder();
        for (int i=0;i<k;i++){
            res.append(words[i]);
            if(i<k-1){
                res.append(" ");
            }
        }
        return res.toString();
    }
}

File: python/strings/Truncate_Sentence_1816.py
class Solution:
    def truncateSentence(self, s: str, k: int) -> str:
        res=[]
        i=1
        words=s.split()
        for word in words:
            if i<=k:
                res.append(word)
            i+=1
        return " ".join(res)